crm_lead_to_task/tests/test_crm_project.py

```diff
@@ -1,24 +1,24 @@
 # Copyright 2021 Tecnativa - Pedro M. Baeza
 # Copyright 2022 Tecnativa - Víctor Martínez
+# Copyright 2024 Tecnativa - Carolina Fernandez
 # License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
 
-from odoo.tests import common
+from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestCrmProject(common.TransactionCase):
+class TestCrmProject(BaseCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         cls.lead = cls.env["crm.lead"].create(
             {
-                "name": "Test lead",
+                "name": "Test Lead",
```

Comment:
Why changing this? Even if we can say that it's the correct upper-casing (which is not true IMO), you shouldn't increase the diff for changes not bringing value to the migration itself.